src/models/user.js

```diff
@@ -1,20 +1,20 @@
 export default (sequelize, DataTypes) => {
   const User = sequelize.define('User', {
-    firstName: {
+    email: {
       allowNull: false,
       type: DataTypes.STRING,
     },
-    lastName: {
-      allowNull: false,
+    firstName: {
+      allowNull: true,
       type: DataTypes.STRING,
     },
-    email: {
-      allowNull: false,
+    lastName: {
+      allowNull: true,
       type: DataTypes.STRING,
       unique: true
     },
     password: {
-      allowNull: false,
+      allowNull: true,
```

Comment:
This should remain `false` since the super admin will have to generate a password before adding the user

Reply:
thanks, would change that